### PR TITLE
Add section on configuring anonymous login for OpenSearch Dashboards

### PR DESCRIPTION
**Pull request by cwperks**
### Description

This PR adds information on how to configure OpenSearch Dashboards with anonymous login. I noticed that the documentation website shows how to see this up for the backend OpenSearch cluster, but does not specify how to configure it for OpenSearch Dashboards.

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
Thank you, @cwperks ! Can you please address the Vale error?

**Comment by cwperks** [comment]
> Can you please address the Vale error?

@hdhalter I merged in the latest from main and see all passing checks. Where can I check for this error?

**Comment by hdhalter** [comment]
> > Can you please address the Vale error?
> 
> @hdhalter I merged in the latest from main and see all passing checks. Where can I check for this error?

Look in the files tab. It's minor - a problem with the capitalization in a heading.

**Comment by cwperks** [comment]
> a problem with the capitalization in a heading.

fixed.

**Review by vagimeli — APPROVED** [review]
Looks great! Sending for editorial review.

File: _security/access-control/anonymous-authentication.md [hunk 1]
```diff
@@ -30,6 +30,19 @@ The following table describes the `anonymous_auth_enabled` setting. For more inf
 If you disable anonymous authentication, you must provide at least one `authc` in order for the Security plugin to initialize successfully.
 {: .important }
 
+## OpenSearch Dashboards configuration
+
+To enable anonymous authentication for OpenSearch Dashboards, you need to modify the `opensearch_dashboards.yml` file inside the configuration directory of your OpenSearch Dashboards installation.
```
Comment (natebower) [inline _security/access-control/anonymous-authentication.md:35]:
```suggestion
To enable anonymous authentication for OpenSearch Dashboards, you need to modify the `opensearch_dashboards.yml` file in the configuration directory of your OpenSearch Dashboards installation.
```

File: _security/access-control/anonymous-authentication.md [hunk 2]
```diff
@@ -30,6 +30,19 @@ The following table describes the `anonymous_auth_enabled` setting. For more inf
...
+
+Add the following setting to `opensearch_dashboards.yml`:
+
+```yml
+opensearch_security.auth.anonymous_auth_enabled: true
+```
+
+Anonymous login for OpenSearch Dashboards requires that anonymous authentication is enabled on the OpenSearch cluster.
```
Comment (natebower) [inline _security/access-control/anonymous-authentication.md:43]:
```suggestion
Anonymous login for OpenSearch Dashboards requires anonymous authentication to be enabled on the OpenSearch cluster.
```

**Review by natebower — COMMENTED** [review]
@cwperks @vagimeli Just a few minor changes. Thanks!

**Pull request merged**